open-hackathon/src/create_test_data.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf8 -*-
 
-'''not necessary for Production environment'''
+"""not necessary for Production environment"""
```

Comment:
could u plz add this file to `gitignore`? I don't want my local test data to be overriden